tutorials/fluid_modeling.ipynb

```diff
@@ -0,0 +1,1267 @@
+{
```

Comment:
<p>Please use punctuation in comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Checked all comments and fixed punctuation.</p>